CodeSniffer/Standards/PSR2/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -68,14 +68,53 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         }
 
         // Only one USE declaration allowed per statement.
-        $next = $phpcsFile->findNext(array(T_COMMA, T_SEMICOLON), ($stackPtr + 1));
-        if ($tokens[$next]['code'] === T_COMMA) {
+        $next = $phpcsFile->findNext(array(T_COMMA, T_SEMICOLON, T_OPEN_USE_GROUP), ($stackPtr + 1));
```

Comment:
Is `T_OPEN_USE_GROUP` created also, when PHP 5 code is parsed OR PHP 5 is used to parse PHP 7 code?